Sources/Stores/NetworkPersistableStore.swift

```diff
@@ -171,7 +169,13 @@ where Network.Remote == Data, Network.Request == URLRequest, Network.Response ==
 
             // update persistence with new value
             if !fromCache {
-                self.persist(payload, for: resource)
+                persistenceStack.setObject(payload, for: resource.persistenceKey) {
+                    switch $0 {
+                    case .success: break
+                    case .failure(let error):
+                        print("⚠️ [Alicerce.NetworkPersistableStore]: Failed to persist value for '\(resource)': \(error)")
```

Comment:
In the future, we should allow the API consumer to provide a custom logger.

Reply:
Agreed, we should revisit this and other (few) cases 👍